consensus/consensus_leader_msg.go

```diff
@@ -46,18 +92,20 @@ func (consensus *Consensus) constructPreparedMessage() ([]byte, *bls.Sign) {
 	// add block content in prepared message for slow validators to catchup
 	consensusMsg.Block = consensus.block
 
-	//// Payload
-	buffer := bytes.NewBuffer([]byte{})
+	// Payload
+	buffer := bytes.Buffer{}
```

Comment:
what's the difference?

Reply:
No need to make a byte which will get garbage collected immediately, NewBuffer already has its own buffer, so lessen the garbage collection pressure